src/SISSL.xml

```diff
@@ -0,0 +1,351 @@
+<SPDX name="Sun Industry Standards Source License v1.1" identifier="SISSL" osi-approved="true">
+  <urls>
+    <url>http://www.openoffice.org/licenses/sissl_license.html</url>
+    <url>http://opensource.org/licenses/SISSL</url>
+  </urls>
+  <header> The contents of this file are subject to the Sun Standards License Version 1.1 (the "License"); You may not
+          use this file except in compliance with the License. You may obtain a copy of the License at 
+    <alt name="LicLocation" match=".+">_______</alt>. Software distributed under the License is distributed on an
+    "AS IS" basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the specific
+    language governing rights and limitations under the License. The Original Code is 
+    <alt name="code" match=".+">_____</alt>. The Initial Developer of the Original Code is: Sun Microsystems, Inc..
+    Portions created by: 
+    <alt name="createdby" match=".+">_____</alt> are Copyright (C): 
+    <alt name="copyright" match=".+">_____</alt> All Rights Reserved. Contributor(s): 
+    <alt name="contributor" match=".+">_____</alt>
+  </header>
+  <license>
+    <title>
+      <p>Sun Industry Standards Source License - Version 1.1</p>
+    </title>
+    <body>
+      <list>
+        <li>
+          <b>1.0</b>
+          <p>DEFINITIONS</p>
+        </li>
+      </list>
+      <list>
```

Comment:
This list should be a sub-list of 1.0. Similar list indentation issues elsewhere. Pull request submitted.